### PR TITLE
JDSOL spec update for 2.0

### PR DESCRIPTION
**Pull request by markt-asf**
Signed-off-by: Mark Thomas <markt@apache.org>

## Specification PR template
When creating a specification project release review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat-spec-x.y.pdf
- [x] Specification HTML in the form of wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [x] For a [Release Review](https://www.eclipse.org/projects/handbook/#release-review), a summary that a Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification. The TCK users guide MUST include the instructions to run the compatible implementations used to validate the release.
Instructions MAY be by reference.
   - [ ] Updated release record
   - [ ] Generated IP Log
   - [ ] Email to PMC
   - [ ] Start release review by emailing EMO
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      N/A - There is no API for DSOL
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/staged-900/jakarta-debugging-tck-2.0.0.zip
- [x] The URL of the compatibility certification request issue:
      https://github.com/eclipse-ee4j/jsp-api/issues/185
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory. 
      N/A There is no API for DSOL

### CODE REVIEW

**Comment by ivargrimstad** [comment]
# Spec Review Checklist

1. Spec PR
  - [ ] PR uses [template](https://github.com/jakartaee/specifications/blob/master/pull_request_template.md)
  - [ ] Directory of form {spec}/x.y
  - [ ] PDF of form {spec}-spec-x.y.pdf ("-spec" preferred but not required)
  - [ ] HTML of form {spec}-spec-x.y.html ("-spec" preferred but not required)
  - [ ] Index page {spec}/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
  - [ ] Index page {spec}/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_index_template.md)
  - [ ] No other files (e.g., no jakarta_ee_logo_schooner_color_stacked_default.png)
  - [ ] Staging repository link of the form https://jakarta.oss.sonatype.org/content/repositories/staging/jakarta/{spec}/jakarta-{spec}-api/x.y.z/
  - [ ] EFTL TCK link of the form http://download.eclipse.org/.../+.zip
  - [ ] Compatibility certification link of the form https://github.com/eclipse-ee4j/{project}/#{issue}
  - [ ] (Optional) Second PR for just apidocs

2. _index.md
  - [ ] Link to spec pdf
  - [ ] Link to spec html
  - [ ] Link to apidocs
  - [ ] Link to final TCK download zip file of the form https://download.eclipse.org/jakartaee/{spec}/x.y/*{spec}-tck-x.y.z.zip
  - [ ] Link to API jar file of the form https://search.maven.org/artifact/jakarta.{spec}/jakarta.{spec}-api/x.y.z/jar
  - [ ] Name of and link to at least one Compatible Implementation

3. javadocs
  - [ ] Footer contains Eclipse copyright and link to license
  - [ ] ESFL license is included, usually as doc-files/speclicense.html
  - [ ] no META-INF directory in PR
  - [ ] javadocs-jar artifact matches apidocs (optional for this release)

4. Spec PDF
  - [ ] Correct spec title
  - [ ] Version number of the form x.y, not x.y.z
  - [ ] Correct Eclipse copyright line
  - [ ] No DRAFT or SNAPSHOT
  - [ ] Correct Logo

5. Spec HTML
  - [ ] Same as PDF

6. TCK zip file
  - [ ] README file (optional for this release)
  - [ ] EFTL license file, preferably named LICENSE.md
  - [ ] User's Guide (or equivalent documentation)
  - [ ] How to test the Compatible Implementation(s) listed in _index.md above with the TCK (may be in UG)

7. TCK User's Guide (or equivalent documentation)
  - [ ] Software requirements listed
  - [ ] Installation and configuration described
  - [ ] How to run tests
  - [ ] Where to file challenges

8. Compatibility certification request
  - [ ] Request follows [template](https://github.com/jakartaee/specification-committee/blob/master/compatibility-certification-request.md)
  - [ ] SHA-256 fingerprint matches staged TCK zip file
  - [ ] Request issue has `certification` label.

9. TCK results summary
  - [ ] Page is hosted by Compatible Implementation project
  - [ ] Includes all information from certification request
  - [ ] Summary includes number of tests passed, failed, errors
  - [ ] SHA-256 fingerprint matches staged TCK zip file on cert request
 
10. Update Jakarta EE API jar
  - [ ] Update the Jakarta EE API jar by submitting a PR to the jakartaee-api project that updates the version number of your API jar file.

**Comment by markt-asf** [comment]
Done.

**Comment by alwin-joseph** [comment]
The TCK was refreshed at https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/staged-900/jakarta-debugging-tck-2.0.0.zip after the changes to improve the documentation. 

We will promote them to  https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/promoted/ once no more changes are required for the tck bundle.

**Review by ivargrimstad — APPROVED** [review]

**Comment by alwin-joseph** [comment]
Please let me know if any more changes needed in the debugging tck. If not I will move the tck to https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/promoted/ .

**Comment by ivargrimstad** [comment]
> Please let me know if any more changes needed in the debugging tck. If not I will move the tck to https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/promoted/ .

LGTM
Please go ahead witih promoting the tck.

**Comment by alwin-joseph** [comment]
> > Please let me know if any more changes needed in the debugging tck. If not I will move the tck to https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/promoted/ .
> 
> LGTM
> Please go ahead witih promoting the tck.

Thanks. debugging tck is now available at https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/promoted/jakarta-debugging-tck-2.0.0.zip

**Comment by markt-asf** [comment]
I need to update the PR for the newly added CCR issue. Should be done shortly...

**Comment by markt-asf** [comment]
CCR info added and PR re-based as well. I think I have done all I can here. Ping this PR or jsp-dev is something else is required.

File: debugging/2.0/_index.md
Comment (ivargrimstad) [inline debugging/2.0/_index.md:15]: This link needs to follow the naming standard, hence be like this: https://download.eclipse.org/jakartaee/debugging/2.0/jakarta-debugging-tck-2.0.0.zip

It does not point to anything right now, but the tck file you link in the checklist will be promoted to this location and signed by the specification committee after the ballot complets.

**Review by ivargrimstad — CHANGES_REQUESTED** [review]

**Comment by ivargrimstad** [comment]
@markt-asf  When the link to the TCK in the _index.md file is fixed (see above), and a label called  `certification` is added to the CCR, this is ready for ballot. Great work!

**Comment by markt-asf** [comment]
Requested changes made. Again, please ping this PR and/or jsp-dev@ any anything further is required.

**Comment by ivargrimstad** [comment]
> Requested changes made. Again, please ping this PR and/or jsp-dev@ any anything further is required.

Great! I will go ahead and initiate the ballot.

**Review by ivargrimstad — APPROVED** [review]

File: debugging/2.0/_index.md
Comment (kwsutter) [inline debugging/2.0/_index.md:15]: The urls for the sig, sha, and pub files need to be filled in.  Not an issue for the ballot, but this needs to be cleaned up before this PR is merged.

File: debugging/2.0/_index.md
Comment (kwsutter) [inline debugging/2.0/_index.md:20]: This url doesn't resolve.  Is it supposed to be https://download.eclipse.org/ee4j/glassfish/weekly/glassfish-6.0.0-SNAPSHOT-2020-10-04.zip?  Same comment applies to the CR and the TCK Results pages.  We should be pointing at a valid binary for the CI.  Again, I won't hold this against the ballot, but it does need to be resolved before this PR gets merged.  Thanks!

**Review by kwsutter — COMMENTED** [review]
These need to be resolved before this PR is merged.

File: debugging/2.0/_index.md
Comment (markt-asf) [reply debugging/2.0/_index.md]: I've added these since we know what they will be in advance.

**Review by ivargrimstad — APPROVED** [review]

**Comment by ivargrimstad** [comment]
* On ballot completion, the specification committee mentor:
 - [ ] adds this final checklist to the main PR.
 - [ ] adds the `approved` label to the PRs, and sends out the Ballot Summary per this [template](#ballot-summary-email-template) to the [public Jakarta EE Specification Committee email list](jakarta.ee-spec@eclipse.org)
 - [ ] calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job. Manually editing the jenkins Build Information will help identify the build (ie. Mail 2.0 or CDI 3.0).
 - [ ] merges the specification (and apidocs) PRs, ensuring the "date:" field in the _index.md file has an appropriate value to allow publishing.
 - [ ] updates the specification page with the ballot results. This is normally done via a separate PR that should be reviewed, approved, and merged.
 - [ ] notifies the EMO of the ballot results by email to [emo@eclipse-foundation.org](emo@eclipse-foundation.org). Just forward the ballot summary note sent earlier to the public Spec Committee email list.
 - [ ] creates an issue in the specification project that includes the following checklist for the specification project team:
 \<include url of Issue here>

**Pull request merged**
